sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -781,14 +779,14 @@
           {
             "name": "id",
             "in": "path",
-            "description": "The ID of the device whose device twin is to be replaced",
+            "description": "The unique identifier of the device whose twin is to be replaced.",
             "required": true,
             "type": "string"
           },
           {
             "name": "deviceTwinInfo",
             "in": "body",
-            "description": "The twin object that will replace the current device twin",
+            "description": "The twin object that will replace the current device twin.",
```

Comment:
It should be sufficient to say "updated twin object"

Reply:
For consistency, I see we are removing object in descriptions elsewhere.